mne/preprocessing/ssp.py

```diff
@@ -308,17 +316,18 @@ def compute_proj_ecg(raw, raw_event=None, tmin=-0.2, tmax=0.4,
         The drop log, if requested.
     """
     raw = raw.copy() if copy else raw
+    new_average = _deprecate_average(average)
```

Comment:
average = _deprecate_average(average) is fine

Then you don't need to change the line below